src/net/rpc.rs

```diff
@@ -163,6 +164,7 @@ impl RPCPeerInfoData {
         chainstate: &StacksChainState,
         peerdb: &PeerDB,
         exit_at_block_height: &Option<&u64>,
+        genesis_chainstate_hash: &String,
```

Comment:
While this arg should probably be `&Sha256Sum` anyways, using `&String` rather than `&str` is usually discouraged because it forces unnecessary clones when you static strs.